backend/lib/endpoints/auth.js

```diff
@@ -112,6 +112,9 @@ async function routes(app) {
       const { email_verified: emailVerified } = auth0User;
       const { payload } = app.jwt.decode(token);
       const userId = payload[authConfig.jwtMongoIdKey];
+      if (!userId) {
+        throw new Error("no mongo_id found in JWT");
```

Comment:
maybe we can access the error for this from ` app.httpErrors`?

Reply:
that's what I firs thad, but if you notice that the whole block is wrapped in `try/catch` it will get caught below and then log the error and return 500...
I also mentioend this in slack backend channel, unlike other loggers the Pino logger doesn't seem to log the errors if they are passed as the second argument so we were doing that wrong